src/auth-service/utils/delete-mobile-user.js

```diff
@@ -0,0 +1,130 @@
+const { getAuth } = require("firebase-admin/auth");
+const admin = require('firebase-admin');
+
+const constants = require("@config/constants");
+
```

Comment:
Could we have this util inside /utils/create-user.js , just to properly manage the number of util files we have?
Out util files are organise and named after "use cases". So delete mobile app user is under the create-USER use case. @BenjaminSsempala 

Thanks again!

Reply:
Thanks @Baalmart 
I've made these changes